txtdirect.go

```diff
@@ -254,11 +248,25 @@ func customResolver(c Config) net.Resolver {
 }
 
 func query(zone string, ctx context.Context, c Config) ([]string, error) {
+	// Removes port from zone
+	if strings.Contains(zone, ":") {
+		zoneSlice := strings.Split(zone, ":")
+		zone = zoneSlice[0]
+	}
+
+	var newZone string
```

Comment:
I think it's unnecessary to declare ``newZone``. Just check if ``zone`` has the ``basezone`` prefix, if yes put the result of ``strings.Join`` in ``zone``. Then there is no need to use ``else`` too.

Reply:
+1